spec/jobs/build_submission_bundle_job_spec.rb

```diff
@@ -15,9 +15,9 @@
       let(:address_valid?) { false }
       let(:address_errors) { "usps error your zip code is a duck" }
 
-      it "transitions the submission into :bundle_failure" do
+      it "transitions the submission into :failed" do
```

Comment:
i decided to remove the :bundle_failure status and transition to just using :failed status, and then we can use the metadata to create distinction between the different types of failures.